articles/quickstart/webapp/laravel/01-login.md

```diff
@@ -118,21 +118,7 @@ AUTH0_CLIENT_ID=${account.clientId}
 AUTH0_CLIENT_SECRET=YOUR_CLIENT_SECRET
 ```
 
-Call `env()` in `laravel-auth0.php` to get the correct setting:
-
-```php
-// config/laravel-auth0.php
-
-return array(
-    // ...
-    'domain'        => env( 'AUTH0_DOMAIN' ),
-    // ...
-    'client_id'     => env( 'AUTH0_CLIENT_ID' ),
-    // ...
-    'client_secret' => env( 'AUTH0_CLIENT_SECRET' ),
-    // ...
-);
-```
+In `laravel-auth0.php`, the global helper `env()` is used to retrieve these values and load them in your Laravel app.
```

Comment:
Should this be `getenv()` instead of `env()`? Looking at the sample code it is calling `getenv()`.

Reply:
`env()` is built into Laravel and `getenv()` is from a package (which isn't needed).